app/controllers/plan_exports_controller.rb

```diff
@@ -46,6 +46,13 @@ def show
       @selected_phase = @plan.phases.order("phases.updated_at DESC")
                              .detect { |p| p.visibility_allowed?(@plan) }
     end
+
+    #ISSUE205: add contributor to cover page
```

Comment:
We should add explanation to the behaviour of the code instead of referencing the issue number of the repository. If we ever change repositories the issue number will no longer make sense.

Reply:
> We should also add tests to make sure all the people listed actually show up on the reports.

Yeah, I was also considering this. I submitted the issue https://github.com/DMPRoadmap/roadmap/issues/3179 to DMPRoadmap because our current way of testing is not tracking the downloaded file, instead, it is just tracking the original page source. 

So far I haven't figured out the best way of asking RSpec to track the content of the 'downloaded file' (i.e. docx, csv, pdf..) because it is part of the user's computer instead of the rails program. I will continue to think about it. Please let me know if you know anything I can refer it.